### ISSUE TITLE
Study TensorFlow and figure out how to create a working Model

### ISSUE THREAD

**Issue by Sohailm25** [acm-projects/To-Toss-or-Not-to-Toss#16]
- [x] Learn basics of TensorFlow (enough to work with it)
- [x] Figure out how to create, train, and call a model
- [x] How to convert a TensorFlow model to tflite

**Comment by Sohailm25** [comment]
I managed to build a model on top of a pre existing MobileNet CNN model. I used a dataset from kaggle, imported their API, and built the model.

I'm currently working on improving the dataset and seeing if other options have better accuracy, including augmentations/dropouts.

Highkey I'm really excited

**Comment by Sohailm25** [comment]
You can export the model and weights straight from colab, and when the time comes, we can simply import it from a .h5 file. it's BEAUTIFUL
